examples/scenes/src/test_scenes.rs

```diff
@@ -1094,6 +1099,32 @@ mod impls {
         }
     }
 
+    pub(super) fn many_clips(scene: &mut Scene, params: &mut SceneParams) {
+        params.resolution = Some(Vec2::new(1000., 1000.));
+        let mut rng = StdRng::seed_from_u64(42);
+        let mut base_tri = BezPath::new();
+        base_tri.move_to((-50.0, 0.0));
+        base_tri.line_to((25.0, -43.3));
+        base_tri.line_to((25.0, 43.3));
+        for y in 0..10 {
+            for x in 0..10 {
+                let translate =
+                    Affine::translate((100. * (x as f64 + 0.5), 100. * (y as f64 + 0.5)));
+                const CLIPS_PER_FILL: usize = 3;
+                for _ in 0..CLIPS_PER_FILL {
+                    let rot = Affine::rotate(rng.gen_range(0.0..PI));
```

Comment:
I believe that the root path is rotationally symmetric every $$\frac{2\pi}{3}$$ radians, but this is doing a rotation every $$\pi$$ radians.

I think that means there's a bias towards the range which is equivalent to $$\[0, \frac{\pi}{3})$$.

Practically, I don't think this matters here, but I'm wondering if this is an intentional choice.

(And of course, my maths understanding could be very wrong :P)

Reply:
Heh, you're right, there is a bias. Not thinking it through carefully, I assumed the symmetry would make it still uniform, and I wanted to make the code concise. I think I'll leave it, in the sprit of kintsugi.

Reply:
We could use $$\tau$$ ([`TAU`](https://doc.rust-lang.org/stable/std/f64/consts/constant.TAU.html)) here to still be fairly concise.

But practically, this is great!